extensions/amp-story/1.0/amp-story-store-service.js

```diff
@@ -569,6 +585,8 @@ export class AmpStoryStoreService {
       [StateProperty.MUTED_STATE]: true,
       [StateProperty.PAGE_ATTACHMENT_STATE]: false,
       [StateProperty.PAGE_HAS_AUDIO_STATE]: false,
+      [StateProperty.PAGE_HAS_CAPTIONS_STATE]: false,
+      [StateProperty.CAPTIONS_STATE]: true,
```

Comment:
(following up from a conversation with @gmajoulet )
Captions are currently on by default. I think it makes sense to keep this field true so the expected UX does not change.

Reply:
We want them on if provided by the publisher (current default), but off when provided by Monti (for now, till we hear back from product). I think this code will be OK for this PR, we can turn them off for Monti later. Eventually we'll unify their defaults

Reply:
Makes some kind of sense but this is a setting that's kept throughout the story, so what would happen if there are videos with publisher captions and videos with cache captions in the same story? We just turn it on by default (as long as there's one track in the story provided by the publisher)?

Reply:
As I understand we are leaving captions `on` by default - even after Monti captions are launched.
@gmajoulet please confirm